### ISSUE TITLE
question about filter dbs.

### ISSUE THREAD

**Issue by xasancakjnc** [hakyimlab/PredictDb-nextflow#1]
Hi, thanks for your comprehensive workflow. But I have several question.

I  use the intron excision ratios  in a group of samples as the "expression file",  and use this pipline to construct the predictive model. However, when I was processing the filter_db procedure, there were only several models retained (zscore_pval < 0.05), 

1. I wonder if it was because of using "ratio" as the input? 
2. Are "models.db" and "models_filtered_signif.db" can be both use to fit the spredixcan?

**Comment by Fnyasimi** [comment]
Yes from the recent update both models can be used to run spredixcan. If you want to use both rerun the pipeline.
